mappings/net/minecraft/entity/SpawnType.mapping

```diff
@@ -1 +0,0 @@
-CLASS aii net/minecraft/entity/SpawnType
```

Comment:
Just the variants need to be deleted -- not the whole mapping :D

Reply:
Oops, didn't notice there was other stuff there...

Reply:
Alright, fixed it.